src/DebugEngineHost/HostLogger.cs

```diff
@@ -3,101 +3,151 @@
 
 using System;
 using System.Collections.Generic;
+using System.Globalization;
 using System.IO;
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
 
 namespace Microsoft.DebugEngineHost
 {
-    public sealed class HostLogger
+    public enum LogLevel
```

Comment:
>public enum LogLevel [](http://example.com/codeflow?start=4&length=20)

It feels like we could probably share this code and `HostLogChannel` between the two DebugEngineHost implementations. Might want to put it in a shared source file so they stay in sync.

Reply:
Would this be a new assembly or just a common `HostLogChannel.cs` and we point at it directly in each project's csproj.

Reply:
The later